src/components/Search/Search.tsx

```diff
@@ -96,7 +95,6 @@ const Search = ({
       e.preventDefault();
       if (value && value.length > 2) {
         const allResultsLink = encodeURLSearchParams(getAllResultsLink(value));
-        showSearch(false);
```

Comment:
same comment about this. I would think we would want the search to close when we hit enter.